Formula/colmap.rb

```diff
@@ -0,0 +1,33 @@
+class Colmap < Formula
+  desc "General-purpose Structure-from-Motion (SfM) and Multi-View Stereo (MVS) pipeline"
+  homepage "https://colmap.github.io"
+  url "https://github.com/colmap/colmap/archive/refs/tags/3.8.tar.gz"
+  sha256 "02288f8f61692fe38049d65608ed832b31246e7792692376afb712fa4cef8775"
+  license "BSD-3-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
+  depends_on "boost"
+  depends_on "ceres-solver"
+  depends_on "cgal"
+  depends_on "eigen"
+  depends_on "flann"
+  depends_on "freeimage"
+  depends_on "gflags"
+  depends_on "glew"
+  depends_on "glog"
+  depends_on "lz4"
+  uses_from_macos "sqlite"
```

Comment:
[My PR](https://github.com/Homebrew/homebrew-core/pull/129705/files) also included `metis`, `qt@5`, and `suite-sparse`. (I didn't have `lz4`.) What is the consequence of omitting these, does this mean that the Qt GUI is not included?

I referred to [this document](https://colmap.github.io/install.html#id2) for the dependencies list.

Reply:
I am not sure i understand your question.

Reply:
How did you derive the list of dependencies? The GUI relies on Qt, doesn't it?